### DISCUSSION TITLE
Add a way to disable implicit global usings from Uno.Extensions via a prop

### DISCUSSION THREAD

**Discussion by MartinZikmund** [unoplatform/uno.extensions#1970] [Ideas]
## What would you like to be added:

Currently implicit global usings are always included in case the project using extensions has them enabled. However, this may not be desirable when the project wants to keep implicit usings, but not Uno.Extensions global usings.

## Why is this needed:

Easier customizability.

## For which Platform:

All

**Comment by nickrandolph** [comment]
Is this something that other libraries do? 
It's easy enough to make this conditional on a property but not sure how discoverable it would be

**Comment by nickrandolph** [comment]
Moved this to a discussion for the timebeing. 

It's easy enough for devs to exclude individual usings if there's a clash. eg
```xml
<ItemGroup>
<Using Remove="Uno.Extensions.Navigation" />
</ItemGroup>
```
